mtags/src/main/scala-2/scala/meta/internal/pc/AutoImportsProvider.scala

```diff
@@ -26,7 +26,7 @@ final class AutoImportsProvider(
     typedTreeAt(pos)
 
     val importPosition = autoImportPosition(pos, params.text())
-    val context = doLocateImportContext(pos)
+    val context = compiler.doLocateImportContext(pos)
```

Comment:
Not needed.

Reply:
Sorry to unintentional change. I will revert.